### DISCUSSION TITLE
How to Built Multi-sig transaction

### DISCUSSION THREAD

**Discussion by bigbenbeer** [MeshJS/mesh#24] [Q&A] [answered]
How can I build a transaction that contains things moving between two wallets?

I want to swop Ada in one wallet with an NFT in another. Is this possible?

**Comment by jinglescode** [comment]
Hi @bigbenbeer, I'm working on a guide you have described. Stay tune. Alternatively, with the current [documentation](https://mesh.martify.io/), you may be able to figure it out too. Nevertheless, I will update again when we have deployed that guide.

**Reply by bigbenbeer** [reply]
Thank you! Will keep an eye out for that!

**Comment by jinglescode** [comment, marked as answer]
[Multi sig guide](https://meshjs.dev/guides/multisig-minting) to show how to mint a token and signed by 2 wallets.
